src/components/MessengerBox.tsx

```diff
@@ -0,0 +1,36 @@
+import UserNav from './UserNav';
+import { useState } from 'react';
+import user from '../data/user.json';
+import message from '../data/message.json';
+import MessengerInput from './MessengerInput';
+import MessageList from './MessageList';
+import { IChat } from '../interface/interface';
+import styled from 'styled-components';
+
+const MessengerBox = () => {
+  // 사용자 나부터 시작
+  const [currentUser, setCurrentUser] = useState(user[0]);
+
+  //현재 메세지를 보내는 사람이 나일 경우 친구로 유저 변경
+  const switchUser = () => {
+    currentUser.userId === 'me'
+      ? setCurrentUser(user[1])
+      : setCurrentUser(user[0]);
+  };
+
+  const [chatList, setChatList] = useState(message);
```

Comment:
```suggestion
  const [chatList, setChatList] = useState(message);
  const [currentUser, setCurrentUser] = useState(user[0]);

  //현재 메세지를 보내는 사람이 나일 경우 친구로 유저 변경
  const switchUser = () => {
    currentUser.userId === 'me'
      ? setCurrentUser(user[1])
      : setCurrentUser(user[0]);
  };

```

별건 아닌데 state 정의는 한 군데에 모아두는게 보기 편하지 않을까,,, 하는 저의 생각입니다:relaxed:

Reply:
맞아요 .. 모아두는걸 습관화해야겠어요 🤣